parser/fsimporter.go

```diff
@@ -77,35 +77,17 @@ func (fs *FSImporter) GetInternalSubnets() []*net.IPNet {
 	return fs.internal
 }
 
-//Run starts the importing
-func (fs *FSImporter) Run() {
-	// track the time spent parsing
-	start := time.Now()
-	fs.res.Log.WithFields(
-		log.Fields{
-			"start_time": start.Format(util.TimeFormat),
-		},
-	).Info("Starting filesystem import. Collecting file details.")
-
+func (fs *FSImporter) CollectFileDetails() []*fpt.IndexedFile {
 	// find all of the potential bro log paths
 	files := readFiles(fs.importFiles, fs.res.Log)
 
 	// hash the files and get their stats
-	indexedFiles := indexFiles(files, fs.indexingThreads, fs.res)
-
-	// if no compatible files for import were found, handle error
-	if !(len(indexedFiles) > 0) {
-		fmt.Println("\n\t[!] No compatible log files found")
-		return
-	}
+	return indexFiles(files, fs.indexingThreads, fs.res)
+}
```

Comment:
After the logging statements are gone and the actual check for files returned was moved to the calling function this ended up being very short. Almost not worth having a separate function for... almost.